package.json

```diff
@@ -35,35 +35,30 @@
   "author": "Peter Staev <peter@tangrasoft.com>",
   "contributors": [
     "Dick Smith (https://github.com/DickSmith)"
-  ],  
+  ],
   "license": "Apache-2.0",
-  
   "peerDependencies": {
-     "tns-core-modules": "^3.0.0"
+    "tns-core-modules": "^3.1.0 || >3.3.0-"
```

Comment:
Again are we using something special from 3.1+? It is best this to remain at 3.0.0 as long as it works with the new new angular.

Reply:
nativescript-angular used the same as peer dependency.